data/other/ldap_scheme.yml

```diff
@@ -28,6 +28,12 @@ parents:
             - organizationalUnit
             - top
 
+    ou=aliases:
```

Comment:
is it enough to create this OU during regen-conf ? (that being said, I could have tested it before asking)